noir/.gitrepo

```diff
@@ -6,7 +6,7 @@
 [subrepo]
 	remote = https://github.com/noir-lang/noir
 	branch = aztec-packages
-	commit = c0826b58272bbba28d6cd25bc07dd5de3cc12e61
+	commit = 114aa6f90f147fe69ff424e881463a65df26c4e6
```

Comment:
This doesn't line up with upstream however I had to fix a merge conflict and commit so that's likely the cause.

Reply:
You mean the commit? it lines up with the tip of aztec-packages branch

Reply:
Ah, I was checking against master. That makes sense.